lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1418,8 +1470,7 @@ void SessionState::get_charging_updates(
         actions_out->push_back(std::move(action));
         break;
       default:
-        MLOG(MWARNING) << "Unexpected action type "
-                       << service_action_type_to_str(action_type) << " for "
+        MLOG(MWARNING) << "Unexpected action type " << action_type << " for "
```

Comment:
We should revert to using `service_action_type_to_str(action_type)` for better readability

Reply:
corrected.